drivers/display/Kconfig.st7789h2

```diff
@@ -0,0 +1,32 @@
+# ST7789H2 display driver configuration options
+
+# Copyright (c) 2021, Jeremy LOCHE
+# SPDX-License-Identifier: Apache-2.0
+
+config ST7789H2
+	bool "ST7789H2 display driver"
+	depends on GPIO
```

Comment:
please default based on `dt_compat_enabled`

Reply:
Could you please point me the right syntax for that ?

Reply:
Done